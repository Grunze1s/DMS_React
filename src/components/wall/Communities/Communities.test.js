import React from 'react';
import Communities from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < Communities {...props}  />)
    )
}
const dummy_data={
    communitiesList:[{title:"title1", image:"image1.png"},
    {title:"title1", image:"image1.png"}]
}
describe('<Communities />', () => {
  

it("Communities render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

