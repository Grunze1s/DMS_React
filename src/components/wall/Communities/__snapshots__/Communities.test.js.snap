// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Communities /> Communities render 1`] = `ShallowWrapper {}`;
